README.md

```diff
@@ -110,6 +110,12 @@ according to your preferences.
   specified in the manifest file will be ignored, and dependency versions will be resolved dynamically instead (this
   feature cannot be enabled when `Strictly match package version` is selected).
 
+- **Gradle** :
+  <br >Set the full path of the Gradle executable, which allows Exhort to locate and execute the `gradle` command to resolve
+  dependencies for Gradle projects.
+  <br >If the path is not provided, your IDE's `PATH` environment will be used to locate the executable.
```

Comment:
This sentence is written in passive voice.  Technical documentation should be written in active voice.  I suggest rewriting this sentence in active voice.  For example:

By not setting a path to the `gradle` binary, IntelliJ uses its default path environment to locate the file.